cocos/scripting/js-bindings/manual/ScriptingCore.cpp

```diff
@@ -925,8 +925,10 @@ bool ScriptingCore::executeScript(JSContext *cx, uint32_t argc, jsval *vp)
 
 bool ScriptingCore::forceGC(JSContext *cx, uint32_t argc, jsval *vp)
 {
+#if CC_TARGET_PLATFORM != CC_PLATFORM_WIN32
```

Comment:
`Temporary disable forceGC on win32`?
If so, why not add some comments for this?